lib/app.rb

```diff
@@ -2,7 +2,39 @@
 require_relative 'simulator'
 
 class App
-  attr_reader :display, :simulator
+  # attr_reader :display, :simulator
+  MESSAGES = {
```

Comment:
All strings that will be printed will be stored in this constant as a hash for reason:
1. Ease of maintainability, where you have all texts in just one area
2. Less usage of magic strings in codebase